README.md

```diff
@@ -1,38 +1,68 @@
-# Getting Started with Create React App
 
+# Djed-Solidity-WebDashboard
+
```

Comment:
Could you add some interesting badges/shield in the beginning of the readme?  Have a look here (https://shields.io/) to see how to do it.

Reply:
OK. Let's leave shields to another PR.